### DISCUSSION TITLE
Any possible way of having a "Week" view in Calendar component?

### DISCUSSION THREAD

**Discussion by jvbs** [ant-design/ant-design#36053] [Ideas]
So by default, `<Calendar/>` component provides a `Month` and `Year` view, is it possible to also have a `Week` view in it?

Something like this where you can view the 4 weeks in a month?
![image](https://user-images.githubusercontent.com/22732699/173612391-502da13c-5b7c-4d8c-85a8-fef520da23ff.png)

**Comment by bombillazo** [comment]
No, its not implemented.
